### PR TITLE
Manuscript plots

### PR DESCRIPTION
**Pull request by jessicaguo**
This code duplicates Tonantzin's plotting of the ACi and AQ curves as well as her code for correcting and plotting the NPQ data. For inclusion in the November MSR, in anticipation of manuscript writing after TT officially starts her new position.

### CODE REVIEW

**Comment by jessicaguo** [comment]
In addition to a table, here are some figures that this PR should produce:
Fig2_NPQ (1)
Fig1_gas_ex_curves (4)

File: figures/02_parameter_table.Rmd [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  pivot_wider(names_from = trt, values_from = c(p.mean, p.se, p.n), names_sep = ".") %>%
+  filter(trait != "g0M", trait != "g1M") %>%
+  bind_rows(ag_sum) %>%
+  mutate(across(2:5, round, 3))
+```
+
+
+## Create table
+
+Format into pleasant output table using the 'gt' package.
```
Comment (KristinaRiemer) [inline figures/02_parameter_table.Rmd:137]: This table didn't really render for me? I might also recommend looking into the kable and kableExtra R packages for creating pretty tables in Rmds.

**Review by KristinaRiemer — APPROVED** [review]
These both knit successfully on the first try! Whoop whoop!

File: figures/02_parameter_table.Rmd (hunk 1 above)
Comment (jessicaguo) [reply figures/02_parameter_table.Rmd:137]: Thanks, will do when the final table is  needed!

**Comment by jessicaguo** [comment]
Modified the .gitignore to suppress all plots in the figures folder. Can remove when ready for publication.

**Pull request merged**